docs/themerc

```diff
@@ -59,6 +59,8 @@ menu.overlap.x: 0
 menu.overlap.y: 0
 menu.width.min: 20
 menu.width.max: 200
+menu.border.width: 1
+menu.border.color: #a8a5a2
```

Comment:
This should always correspond to the default value (via inheritance or explicitly set)

Reply:
Thanks. Fixed.